apify-api/openapi/paths/actor-tasks/actor-tasks@{actorTaskId}@run-sync.yaml

```diff
@@ -55,11 +55,11 @@ get:
     - name: maxItems
       in: query
       description: |
-        The maximum number of items that the task run should return. This is
-        useful for pay-per-result tasks, as it allows you to limit the number of
-        results that will be charged to your subscription. You can access the
-        maximum number of items in your Actor by using the
-        `ACTOR_MAX_PAID_DATASET_ITEMS` environment variable.
+        Specifies the maximum number of dataset items that will be charged for pay-per-result Actors.
```

Comment:
I see quite a bit of repetition here - could you make some reusable parameter objects and reference them in the relevant endpoints? See https://swagger.io/specification/#components-object